lib/ansible/modules/files/acl.py

```diff
@@ -373,10 +368,12 @@ def main():
 
     acl = run_acl(
         module,
-        build_command(module, 'get', path, follow, default, recursive)
+        build_command(module, 'get', path, follow, default, recursive, no_mask)
     )
 
     module.exit_json(changed=changed, msg=msg, acl=acl)
 
-if __name__ == '__main__':
-    main()
+# import module snippets
+from ansible.module_utils.basic import *
```

Comment:
`from ansible.module_utils.basic import *` is deprecated now.

See http://docs.ansible.com/ansible/dev_guide/developing_modules_checklist.html

> Import ansible.module_utils code in the same place as you import other libraries. In older code, this was done at the bottom of the file but that’s no longer needed.